File: Arrays/2011. Final Value of Variable After Performing Operations.py
def finalValueAfterOperations(operate):
    x = 0
    for i in range(len(operate)):
        if(operate[i]=="++X" or "X++"):
            x = x + 1;
        elif(operate[i]=="--X" or "X--"):
            x = x - 1;
    
    return x


print(finalValueAfterOperations(["++X","++X","X++"]))

File: Arrays/2011. Final Value of Variable After Performing Operations.js
link = 'https://leetcode.com/problems/final-value-of-variable-after-performing-operations/'


// Solution
//---------------------------------
const finalValueAfterOperations = (operations) => {
    let x = 0
    for (let i = 0; i < operations.length; i++) {
        if(operations[i]==='X++' || operations[i]==='++X'){
            x = x + 1
        }else if(operations[i]==='X--' || operations[i]==='--X'){
            x = x - 1
        }
    }
    return x
}
//---------------------------------
let result = finalValueAfterOperations(["--X","X++","X++"])
console.log(result)